Tests/ASScrollNodeTests.m

```diff
@@ -66,13 +67,13 @@ - (void)testAutomaticallyManagesContentSizeUnderflow
   CGSize parentSize = CGSizeMake(100, 200);
   ASSizeRange sizeRange = ASSizeRangeUnconstrained;
 
-  subnode.style.preferredSize = subnodeSize;
+  self.subnode.style.preferredSize = subnodeSize;
 
-  ASLayout *layout = [scrollNode layoutThatFits:sizeRange parentSize:parentSize];
-  [scrollNode layout];
+  ASLayout *layout = [self.scrollNode layoutThatFits:sizeRange parentSize:parentSize];
+  [self.scrollNode layout];
```

Comment:
It seems better (perhaps) to call [scrollNode.layer layoutIfNeeded], to test the full layout stack as it is triggered in most apps?